microbench/src/main/java/io/netty/microbench/headers/HeadersBenchmark.java

```diff
@@ -139,4 +163,546 @@ public void http2Iterate(Blackhole bh) {
             bh.consume(entry);
         }
     }
+
+    @Benchmark
+    @BenchmarkMode(Mode.AverageTime)
+    public void httpAddAllFastest(Blackhole bh) {
+        bh.consume(new DefaultHttpHeaders(false).add(httpHeaders));
```

Comment:
I think we should create the DefaultHttpHeaders as part of setup as otherwise these will also influence the benchmark. Same is true for other methods.